lib/util/exec/utils.ts

```diff
@@ -21,6 +22,7 @@ export function getChildEnv({
     ...extraEnv,
     ...parentEnv,
     ...globalConfigEnv,
+    ...userConfiguredEnv,
```

Comment:
why, they are already in `extraEnv`?

Reply:
`extraEnv` is not passed in the `getChildEnv` function, and is only used to generate a list of `envVars`.